### ISSUE TITLE
ADDIE crashing - IPython Dock Widget

### ISSUE THREAD

**Issue by marshallmcdonnell** [neutrons/addie#99]
ADDIE is crashing due to the IPython Dock Widgets
```
/usr/lib/python2.7/site-packages/IPython/utils/traitlets.py:5: UserWarning: IPython.utils.traitlets has moved to a top-level traitlets package.
  warn("IPython.utils.traitlets has moved to a top-level traitlets package.")
/usr/lib/python2.7/site-packages/IPython/kernel/__init__.py:13: ShimWarning: The `IPython.kernel` package has been deprecated since IPython 4.0.You should import from ipykernel or jupyter_client instead.
  "You should import from ipykernel or jupyter_client instead.", ShimWarning)
Traceback (most recent call last):
  File "build/scripts-2.7/addie", line 2266, in <module>
    main()
  File "build/scripts-2.7/addie", line 2261, in main
    form = MainWindow()
  File "build/scripts-2.7/addie", line 125, in __init__
    self.ui.dockWidget_ipython.setup()
  File "/SNS/users/ntm/software/addie/build/lib/addie/ipythondockwidget.py", line 21, in setup
    self.iPythonWidget = mantid_ipython_widget.MantidIPythonWidget()
  File "/SNS/users/ntm/software/addie/build/lib/addie/mantid_ipython_widget.py", line 66, in __init__
    super(MantidIPythonWidget, self).__init__(*args, **kw)
  File "/usr/lib/python2.7/site-packages/IPython/qt/console/rich_ipython_widget.py", line 39, in __init__
    super(RichIPythonWidget, self).__init__(*args, **kw)
  File "/usr/lib/python2.7/site-packages/IPython/qt/console/ipython_widget.py", line 118, in __init__
    super(IPythonWidget, self).__init__(*args, **kw)
  File "/usr/lib/python2.7/site-packages/IPython/qt/console/frontend_widget.py", line 155, in __init__
    super(FrontendWidget, self).__init__(*args, **kw)
  File "/usr/lib/python2.7/site-packages/IPython/qt/console/history_console_widget.py", line 27, in __init__
    super(HistoryConsoleWidget, self).__init__(*args, **kw)
  File "/usr/lib/python2.7/site-packages/IPython/qt/console/console_widget.py", line 245, in __init__
    LoggingConfigurable.__init__(self, **kw)
  File "/usr/lib/python2.7/site-packages/IPython/config/configurable.py", line 83, in __init__
    self._load_config(self.config)
  File "/usr/lib/python2.7/site-packages/IPython/config/configurable.py", line 132, in _load_config
    my_config = self._find_my_config(cfg)
  File "/usr/lib/python2.7/site-packages/IPython/config/configurable.py", line 113, in _find_my_config
    if self.parent:
  File "/usr/lib/python2.7/site-packages/traitlets/traitlets.py", line 556, in __get__
    return self.get(obj, cls)
  File "/usr/lib/python2.7/site-packages/traitlets/traitlets.py", line 535, in get
    value = self._validate(obj, dynamic_default())
  File "/usr/lib/python2.7/site-packages/traitlets/traitlets.py", line 591, in _validate
    value = self.validate(obj, value)
  File "/usr/lib/python2.7/site-packages/traitlets/traitlets.py", line 1677, in validate
    self.error(obj, value)
  File "/usr/lib/python2.7/site-packages/traitlets/traitlets.py", line 1524, in error
    raise TraitError(e)
traitlets.traitlets.TraitError: The 'parent' trait of a MantidIPythonWidget instance must be a Configurable, but a value of type 'NoneType' (i.e. None) was specified.
```

**Comment by marshallmcdonnell** [comment]
The PyQt5 vs. PyQt4 bindings still break ADDIE on analysis.sns.gov

**Comment by marshallmcdonnell** [comment]
Closed in 4810effa20dd40fe76a51ef5e50215dd9e2389ec of master
